route/build/src/read_streamSeg.f90

```diff
@@ -376,149 +494,149 @@ subroutine getData(&
-            case(3); HYPE_is_called     = .true.; number_HYPE     = number_HYPE     + 1; ! add number of HYPE lakes
-            case default; ierr=20; message=trim(message)//'unable to identify the lake model type'; return
-          end select
-        else ! if for a lake both parameteric and non parameteric are set to correct
-          number_TargVol = number_TargVol + 1 ! add number of target volume case
-          select case(LakeModelType_local(i))
-            case(1); ierr=20; message=trim(message)//'both data driven (follow target volume) and Doll lake formulation are activated for a lake'; return
-            case(2); ierr=20; message=trim(message)//'both data driven (follow target volume) and Hanasaki lake formulation are activated for a lake'; return
-            case(3); ierr=20; message=trim(message)//'both data driven (follow target volume) and HYPY lake formulation are activated for a lake'; return
-          end select
-        endif
-      endif
-    enddo
-
-    ! check the local lake model flags with the global flags
-    if ((Doll_is_called).and.(.not.(lake_model_D03))) then
-      ierr=20; message=trim(message)//'Doll 2003 is called in the netwrok topology but the flag is not set to true in control file; set the flag to true in control file'; return
-    endif
-
-    ! check the local lake model type with the global flags
-    if ((Hanasaki_is_called).and.(.not.(lake_model_H06))) then
-      ierr=20; message=trim(message)//'Hanasaki 2006 is called in the netwrok topology but the flag is not set to true in control file; set the flag to true in control file'; return
-    endif
-
-    ! check the local lake model type with the global flags
-    if ((HYPE_is_called).and.(.not.(lake_model_HYPE))) then
-      ierr=20; message=trim(message)//'HYPE is called in the netwrok topology but the flag is not set to true in control file; set the flag to true in control file'; return
-    endif
-
-    ! print the numbers
-    print*, "total number of lakes             = ", number_lakes
-    print*, "lakes with Doll formulation       = ", number_Doll
-    print*, "lakes with Hanasaki formulation   = ", number_Hanasaki
-    print*, "lakes with HYPE formulation       = ", number_HYPE
-    print*, "lakes with target volume          = ", number_TargVol
-
-    ! check is the number of parameteric lakes and target volume sums up to the total number of lakes
-    if (number_Doll+number_Hanasaki+number_HYPE+number_TargVol == number_lakes) then
-      print*, "number of lake models and target volume models matches the total number of lakes; should be good to go!"
-    else
-      ! print*, "number of lake models and target volume models do not match the total number of lakes"
-      ierr=20; message=trim(message)//'number of lake models and target volume models do not match the total number of lakes'; return
-    endif
-  endif
+!  ! ---------- check the consistency of the lake models  ----------------------------------------------------------
```

Comment:
These lines commented out are needed?   if not any longer, maybe remove them?

Reply:
The commented block is not needed and was removed.